src/main/javascript/software.bytepushers.utils.DateUtility.js

```diff
@@ -27,5 +27,28 @@
         }
         return 0;
     };
+    /*checks if string passed in is valid MM/DD/YYYY date*/
+    BytePushers.DateUtility.isDateString_MMDDYYYY = function (dateString) {
```

Comment:
Probably should rename method name to validateDateFormat and pass in a date format constant MMDDYY and then use a switch statement to delegate to various date formate validators to ensure the dateString is formatted correctly.  If we refactor it this way we can expand support more date format validation in the future.